playground/backend/cmd/server/controller.go

```diff
@@ -123,35 +124,61 @@ func (controller *playgroundController) GetCompileOutput(ctx context.Context, in
 
 //GetListOfExamples returns the list of examples
 func (controller *playgroundController) GetListOfExamples(ctx context.Context, info *pb.GetListOfExamplesRequest) (*pb.GetListOfExamplesResponse, error) {
-	// TODO implement this method
-	example1 := pb.Example{ExampleUuid: "001", Name: "Example1", Description: "Test example 1", Type: pb.ExampleType_EXAMPLE_TYPE_DEFAULT}
-	example2 := pb.Example{ExampleUuid: "003", Name: "Example3", Description: "Test example 3", Type: pb.ExampleType_EXAMPLE_TYPE_KATA}
-
-	cat1 := pb.Categories_Category{
-		CategoryName: "Common",
-		Examples:     []*pb.Example{&example1, {ExampleUuid: "002", Name: "Example2", Description: "Test example 1", Type: pb.ExampleType_EXAMPLE_TYPE_UNIT_TEST}},
+	cd := storage.NewCloudStorage()
+	examples, err := cd.GetListOfExamples(ctx, info.Sdk.String(), info.Category)
+	if err != nil {
+		logger.Errorf("%s: GetListOfExamples(): cloud storage error: %s", err.Error())
+		return nil, errors.InternalError("GetListOfExamples(): ", err.Error())
 	}
-	cat2 := pb.Categories_Category{
-		CategoryName: "I/O",
-		Examples:     []*pb.Example{&example2},
+	response := pb.GetListOfExamplesResponse{SdkExamples: make([]*pb.Categories, 0)}
+	for sdk, categories := range *examples {
```

Comment:
In my practice, if you can write 3rd nested loop, you can try to avoid it. Let's think about how to design it better.